interop_binaries/src/bin/janus_interop_collector.rs

```diff
@@ -675,40 +675,39 @@ fn make_filter() -> anyhow::Result<impl Filter<Extract = (Response,)> + Clone> {
             }
         }
     });
-    let collect_start_filter =
-        warp::path!("collect_start").and(warp::body::json()).then({
+    let collect_start_filter = warp::path!("collect_start").and(warp::body::json()).then({
```

Comment:
Q: should these be renamed? (to e.g. `collection_start`)

Doesn't need to happen in this PR, could be included as part of test API updates for DAP-04.

Reply:
I think it's reasonable for this to be the verb to collect rather than the noun collection. If anyone disagrees the place to start is a change to the interop test spec.